firmware/application/external/pacman/playfield.hpp

```diff
@@ -1243,21 +1264,41 @@ class Playfield {
     }
 
     void Step() {
-        int16_t keys = 0;
-
         if (GAMEWIN == 1) {
+            cheat_level = false;
+            cheat_lifes = false;
             LEVEL++;
             Init();
         }
 
         // Start GAME
-        if (but_A && DEMO == 1 && GAMEPAUSED == 0) {
+        if (but_A && DEMO == 1 && GAMEPAUSED == 0) {  // start
             but_A = false;
-            DEMO = 0;
             Init();
-        } else if (but_A && DEMO == 0 && GAMEPAUSED == 0) {  // Or PAUSE GAME
+            DEMO = 0;
+        } else if (but_A && DEMO == 0 && GAMEPAUSED == 0) {  // pause
             but_A = false;
             GAMEPAUSED = 1;
+        } else if (but_LEFT && DEMO == 1 && GAMEPAUSED == 0) {  // -level
+            cheat_level = true;
+            but_LEFT = false;
+            LEVEL--;
```

Comment:
Are we OK with this wrapping to 255?

Reply:
ahhh sorry i didn't consider this, it's working good and returned to 1,  but considering the rigor i'll fix this one

Reply:
fixed at [`eb90290` (#1821)](https://github.com/portapack-mayhem/mayhem-firmware/pull/1821/commits/eb90290129fe6522d85963afe81cf9f88afd37d3)